modules/proto/rollkitsdk/network/module/v1/module.proto

```
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package rollkitsdk.network.module.v1;
+
+import "cosmos/app/v1alpha1/module.proto";
```

Buf (1.54.0)

import "cosmos/app/v1alpha1/module.proto": file does not exist

(COMPILE)